website/views/static/press-kit.jade

```diff
@@ -19,7 +19,32 @@ block content
   p= env.t('presskitDownload')
     | &nbsp;
     a.btn.btn-success(href='/presskit/presskit.zip') presskit.zip
-  - var imgs = ['Habitica Text Logo', 'Habitica Logo Promo', 'Habitica Gryphon', 'Dread Drag\'on of Dilatory', 'Market Sample Screen', 'Challenges Sample Screen', 'Equipment Sample Screen', 'Guilds Sample Screen', 'HabiticaPromoThin', 'Laundromancer', 'Tasks Page', 'SnackLess Monster', 'Battling the Ghost Stag', 'Stagnant Dishes Boss', 'Necro-Vice']
+  - var imgs = ['Habitica Promo',
```

Comment:
Oh. I will do that, and will commit it.
On Mon, Mar 14, 2016 at 10:29 PM Blade Barringer <notifications@github.com>
wrote:

> In website/views/static/press-kit.jade
> <https://github.com/HabitRPG/habitrpg/pull/6846#discussion_r56111190>:
>
> > @@ -19,7 +19,32 @@ block content
> >    p= env.t('presskitDownload')
> >      | &nbsp;
> >      a.btn.btn-success(href='/presskit/presskit.zip') presskit.zip
> > -  - var imgs = ['Habitica Text Logo', 'Habitica Logo Promo', 'Habitica Gryphon', 'Dread Drag\'on of Dilatory', 'Market Sample Screen', 'Challenges Sample Screen', 'Equipment Sample Screen', 'Guilds Sample Screen', 'HabiticaPromoThin', 'Laundromancer', 'Tasks Page', 'SnackLess Monster', 'Battling the Ghost Stag', 'Stagnant Dishes Boss', 'Necro-Vice']
> > +  - var imgs = ['Habitica Promo',
>
> That SO question should be correct. You need the dash on a line by itself
> and the variable declaration declared one indent in from that. I believe
> it's like this:
>
> -  var images = ['Habitica Promo',    'Habitica Promo - Thin',     // etc
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/HabitRPG/habitrpg/pull/6846/files#r56111190
>